code/drasil-example/gamephysics/lib/Drasil/GamePhysics/GenDefs.hs

```diff
@@ -75,8 +75,8 @@ accelGravityDesc = foldlSent [S "If one of the", plural QPP.mass, S "is much lar
   phrase QP.force]
 
 accelGravityExpr :: PExpr
-accelGravityExpr = neg ((sy QP.gravitationalConst `mulRe` sy mLarger $/
-  square (sy dispNorm)) `mulRe` sy dVect)
```

Comment:
Jason is this what you are referring to regarding inlining?

Reply:
Yep! Sorry, I meant to say to switch to using the "infix" notation, not inlining! This example is a good one, where you changed the old infix notation to the new infix notation. There are just some that are of the form `mulRe A B`, which should be `A $* B` instead. I think you changed these now, though?

Reply:
I just did now

Reply:
Thanks!